modules/datahandling/requestmap_test.go

```diff
@@ -23,11 +27,8 @@ func TestProjectCreateRequest(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	switch newRequest.(type) {
-	default:
-		t.Fatalf("newRequest is the wrong type, expected: *projectCreateRequest, actual: %s\n", reflect.TypeOf(newRequest))
-	case *projectCreateRequest:
-
+	if reflect.TypeOf(newRequest).String() != "*datahandling.projectCreateRequest" {
```

Comment:
Go convention is to avoid using reflection, even in tests. We might want to think about adding a type field or method to the structs. Perhaps put in a todo to clean up these tests (and any others that use reflection?)